dashboard-generator/generator.js

```diff
@@ -134,14 +135,18 @@ window.addEventListener('load', function () {
       _dash.splice(plotIndex, 1);
       updatePostData();
       tr.parentElement.removeChild(tr);
+      countDash--;
+      if(countDash==0)
+        $('#generate').setAttribute('disabled',true);
```

Comment:
In this file `if` statements are all at least three lines long, requiring `{}` around their bodies.